ui/src/app/features/model-table/model-table.component.spec.ts

```diff
@@ -13,17 +13,16 @@
  * or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatTableModule } from '@angular/material/table';
-import { MatIconModule } from '@angular/material/icon';
-
-import { ModelTableComponent } from './model-table.component';
-import { TranslatePipe, TranslateService } from '@ngx-translate/core';
 import { MatTooltipModule } from '@angular/material/tooltip';
+import { TranslatePipe, TranslateService } from '@ngx-translate/core';
+
 import { TruncateModule } from '../../ui/truncate-pipe/truncate.module';
+import { ModelTableComponent } from './model-table.component';
 
 xdescribe('ModelTableComponent', () => {
```

Comment:
It looks like that test was excluded

Reply:
good spot, not part of the changes in this PR though. let's fix those tests in another PR